cmd/compute-domain-controller/computedomain.go

```diff
@@ -181,13 +185,13 @@ func (m *ComputeDomainManager) RemoveFinalizer(ctx context.Context, uid string)
 		}
 	}
 	if len(cd.Finalizers) == len(newCD.Finalizers) {
+		klog.Infof("Finalizer not found on ComputeDomain %s/%s, nothing to do", cd.Namespace, cd.Name)
```

Comment:
This should be (at least) verbose level 2.

Reply:
Noted.